docs/reference/commandline/images.md

```diff
@@ -155,3 +155,24 @@ Ready for use by `docker rmi ...`, like:
 NOTE: Docker will warn you if any containers exist that are using these untagged images.
 
 
+##### Labelled images
```

Comment:
s/Labelled/Labeled/

At least; that's what this says (looked it up to be sure); http://grammarist.com/spelling/label/ :smile:

> In American English, <em>label </em>makes <strong><em>labeled </em></strong>and <em>labeling</em>, with one <em>l</em>. In Canada and in varieties of English from outside North America, the preferred forms are <strong><em>labelled </em></strong><em>labelling</em>...